webapp/components/wpt-results.html

```diff
@@ -131,8 +131,10 @@
     <template is="dom-if" if="{{ pathIsATestFile }}">
       <div class="links">
         <ul>
-          <li><a href$="https://github.com/w3c/web-platform-tests/blob/master[[path]]" target="_blank">View source on GitHub</a></li>
-          <li><a href$="[[scheme]]://w3c-test.org[[path]]" target="_blank">Run in your browser on w3c-test.org</a></li>
+          <li><a href$="https://github.com/w3c/web-platform-tests/blob/master[[sourcePath]]" target="_blank">View source on GitHub</a></li>
```

Comment:
Aside: Would it be straightforward to use the SHA here to get to exactly the right version? Especially when looking at old results the chance for confusion is otherwise there. (Can't fix it for w3c-test.org of course.)

Reply:
Yes, it would; I'll follow up this PR with that!